customer_outstanding_statement/README.rst

```diff
@@ -0,0 +1,72 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================================
+Print Customer Outstanding Statement
+====================================
+
+The outstanding statement provides details of all outstanding customer receivables
+up to a particular date. This includes all unpaid invoices, unclaimed refunds and
+outstanding payments. The list is displayed in chronological order and is split by currencies.
+
+Aging details can be shown in the report, expressed in aging buckets (30 days
+due, ...), so the customer can review how much is open, due or overdue.
+
```

Comment:
The Due Payments module (<code>account.report_overdue</code>) you say is printing all activity on the customer receivables from all the history of the customer, nothing related with the Customer Outstanding Statement that just prints the outstanding. The Customer Activity Statement (https://github.com/OCA/account-financial-reporting/pull/290) is more related to the Due Payments.